### DISCUSSION TITLE
API returns no future dates

### DISCUSSION THREAD

**Discussion by BeSt1979** [vasqued2/ha-teamtracker#179] [General]
Hi. I try to get upcoming games of my most liked German football team, but the API returns only a small time span in the past:

attribution: Data provided by ESPN
sport: soccer
league: BUND
league_logo: https://a.espncdn.com/i/leaguelogos/soccer/500/10.png
team_abbr: "134"
opponent_abbr: null
event_name: null
event_url: null
date: null
kickoff_in: null
series_summary: null
venue: null
location: null
tv_network: null
odds: null
overunder: null
team_name: null
team_id: null
team_record: null
team_rank: null
team_homeaway: null
team_logo: null
team_url: null
team_colors: null
team_score: null
team_win_probability: null
team_winner: null
team_timeouts: null
opponent_name: null
opponent_id: null
opponent_record: null
opponent_rank: null
opponent_homeaway: null
opponent_logo: null
opponent_url: null
opponent_colors: null
opponent_score: null
opponent_win_probability: null
opponent_winner: null
opponent_timeouts: null
quarter: null
clock: null
possession: null
last_play: null
down_distance_text: null
outs: null
balls: null
strikes: null
on_first: null
on_second: null
on_third: null
team_shots_on_target: null
team_total_shots: null
opponent_shots_on_target: null
opponent_total_shots: null
team_sets_won: null
opponent_sets_won: null
last_update: "2024-09-07 01:00:50+02:00"
api_message: >-
  No competition scheduled for '134' between 2024-09-01T13:30Z and
  2024-09-01T15:30Z
icon: mdi:soccer
friendly_name: vfb

**Comment by vasqued2** [comment]
It's a FIFA break.  There are no games until mid-month.  The API will return games once you get within about 5 days of the next game.

**Comment by BeSt1979** [comment]
Aaaaaaaa - I wasn't wäre, that there ist No Game this Weekend.

Thank you for your Help.

David ***@***.***> schrieb am Sa., 7. Sept. 2024, 02:03:

**Comment by francescomadeo** [comment]
why is he answering me like this? if I enter JUV or another team from the Italian championship it responds correctly

attribution: Data provided by ESPN
sport: soccer
league: SERA
league_logo: https://a.espncdn.com/i/leaguelogos/soccer/500/12.png
team_abbr: MIL
opponent_abbr: null
event_name: null
event_url: null
date: null
kickoff_in: null
series_summary: null
venue: null
location: null
tv_network: null
odds: null
overunder: null
team_name: null
team_id: null
team_record: null
team_rank: null
team_homeaway: null
team_logo: null
team_url: null
team_colors: null
team_score: null
team_win_probability: null
team_winner: null
team_timeouts: null
opponent_name: null
opponent_id: null
opponent_record: null
opponent_rank: null
opponent_homeaway: null
opponent_logo: null
opponent_url: null
opponent_colors: null
opponent_score: null
opponent_win_probability: null
opponent_winner: null
opponent_timeouts: null
quarter: null
clock: null
possession: null
last_play: null
down_distance_text: null
outs: null
balls: null
strikes: null
on_first: null
on_second: null
on_third: null
team_shots_on_target: null
team_total_shots: null
opponent_shots_on_target: null
opponent_total_shots: null
team_sets_won: null
opponent_sets_won: null
last_update: "2024-09-07 09:46:53+02:00"
api_message: >-
  No competition scheduled for 'MIL' between 2024-09-01T16:30Z and
  2024-09-01T18:45Z
icon: mdi:soccer
friendly_name: team_tracker_serie_a

**Reply by vasqued2** [reply]
Same answer as above.  It's a FIFA break. There are no games until mid-month. The API will return games once you get within about 5 days of the next game.

**Reply by francescomadeo** [reply]
forgive me, but if I search for the Juventus team of the Italian championship it comes out correctly, try it too and you will see that there is a bug.

**Reply by vasqued2** [reply]
There are no Series A games scheduled until Sept 14 due to the FIFA break.  I can't find any Juventus games when I search.  Are you talking about the women's team?  Please provide a link to what you are talking about.

It's possible Juventus has a friendly or a tournament game but you would need to set up a separate sensor for that since they wouldn't be included in the Series A tracker you set up.

**Comment by Waank1** [comment]
Now that the regular season is over for MLB I am working on some of my automations.  What attribute tells me game start time?  Is kickoff_in: applicable to baseball?  If so I can derive start time form that?

TIA

**Comment by vasqued2** [comment]
`date` has the date and start time.  You can also use `kickoff_in` for automations if you prefer.  It works for all sports and its name is a carryover from the original nfl sensor.

**Reply by Waank1** [reply]
Thanks!
